stable/nginx-ingress/README.md

```diff
@@ -64,6 +64,7 @@ Parameter | Description | Default
 `controller.service.externalIPs` | controller service external IP addresses | `[]`
 `controller.service.loadBalancerIP` | IP address to assign to load balancer (if supported) | `""`
 `controller.service.loadBalancerSourceRanges` | list of IP CIDRs allowed access to load balancer (if supported) | `[]`
+`controller.service.loadBalancerTerminateSSL` | terminates SSL at load balancer (forwards 443 to 80) | false
```

Comment:
I also think the naming is misleading. I understood the opposite of what you meant. Effectively, this just changes the target port, so why no simply make that configurable?

`controller.service.targetPort` with a default of 443 instead of `controller.service.loadBalancerTerminateSSL`.